docs/batch_inference/test_and_deploy_batch_model_inference.md

```diff
@@ -0,0 +1,39 @@
+(test_and_deploy_batch_model_inference)=
+
+# Test and deploy batch model inference
+
+## Testing model
+
+To evaluate batch model inference, you should use the evaluate handler of the `auto_trainer` function.
+
+This is typically done during model development, for more information refer to the {ref}`auto_trainer_evaluate` handler documentation. For example:
```

Comment:
This is typically done during model development. For more information refer to the {ref}`auto_trainer_evaluate` handler documentation. For example: